db/db_impl/db_impl.cc

```diff
@@ -2042,6 +2042,50 @@ Status DBImpl::GetEntity(const ReadOptions& _read_options,
   return GetImpl(read_options, key, get_impl_options);
 }
 
+void DBImpl::GetEntity(const ReadOptions& _read_options, const Slice& key,
+                       PinnableAttributeGroups& result) {
+  if (result.size() == 0) {
```

Comment:
Another way to handle this would be adding a default column family to the result if nothing is provided, but not too sure if that's any less confusing or better for the users.